Fits/Interceptors/[ADV-5] Taranis.md

```diff
@@ -0,0 +1,41 @@
+# Taranis
+
+High DPS Interceptor 
+
+
+## Description
+
+A very fast Interceptor focued on burning pings and repositioning, with DPS to target/catch/kill enemy EWAR frigates and long range disruption.
+
+## Fit
+
+```
+[Taranis, [ADV-5] Taranis]
+
+Damage Control II
+Reinforced Bulkheads II
+Vigor Compact Micro Auxiliary Power Core
+
+5MN Quad LiF Restrained Microwarpdrive
+1MN Afterburner II
```

Comment:
AB doesn't necessarily seem worth it?

Reply:
Honestly idk what to do with the Taranis, I think dual prop DPS is probably it's best use because with it's slot layout it just becomes paper if you do even 1x overdrive.